skstats/hypotests/calculators/asymptotic_calculator.py

```diff
@@ -0,0 +1,337 @@
+from .basecalculator import BaseCalculator
+from ..fit_utils.utils import eval_pdf, array2dataset, pll
+from ..parameters import POI
```

Comment:
Picky suggestion: I find it nice to organise import statements from standard libs to local to the package. Here I would write

```
import numpy as np
from scipy.stats import norm

from ..parameters import POI
```

Note the space between std and local stuff.

Same goes for all the below.

Reply:
It looks tidier indeed.